index.rst

```diff
@@ -1,51 +1,1166 @@
 ..
   Technote content.
 
-  See https://developer.lsst.io/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. _scope:
 
-  Use the following syntax for sections:
+Scope of the document
+=====================
 
-  Sections
-  ========
+This document is designed to assist developers involved in porting LSST Science Pipelines
+from swig to pybind11.
 
-  and
+.. _intro:
 
-  Subsections
-  -----------
+.. _installation:
 
-  and
+Additional documentation
+========================
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+Also see the:
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+* `The pybind11 upstream documentation <http://pybind11.readthedocs.io>`_ 
+* `The LSST pybind11 coding guidelines <https://dmtn-024.lsst.io>`_ 
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Installation
+============
 
-     Caption text.
+To install all of the currently wrapped code with lsstsw, use
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+.. code-block:: bash
 
-   Feel free to delete this instructional comment.
+    rebuild -r tickets/DM-6168 {{package name}}
 
-:tocdepth: 1
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+where ``{{package name}}`` is the name of the package that is currently being wrapped (for instance ``afw``).
+This will build the most up to date version of the stack that has been ported to pybind11, as the tests that have not been wrapped are all commented out (see section :ref:`activate_test`).
+
+Don't forget to tag the new build as current with EUPS:
+
+.. code-block:: bash
+
+    eups tags --clone bNNNN current
+
+where ``bNNN`` is the current build number.
+
+If you plan to collaborate with multiple people on the same ticket you will also need to install `gitlock`_ to allow you to get a soft lock on files that you are working on.
```

Comment:
Originally I think the pybind11 port was supposed to be finished in the current sprint, so we thought there could be as many as 10 other developers working on afw at the same time. That was the main reason I developed gitlock. With only 3-4 developers it turns out that frequent communication on a slack channel was much more useful. :smile: I'll remove this part of the tutorial.